### DISCUSSION TITLE
melos bootstrap command not working

### DISCUSSION THREAD

**Discussion by Killpit** [invertase/melos#785] [Q&A]
I try to bootstrap melos but I get command not found, I installed melos, configured it through the shell, saved the cache to my computer and saved it to shell, checked its version, restarted terminal and did direct invocation to get the command not found error. Need help to solve the problem and I'm the first time user

**Comment by spydon** [comment]
What operating system are you on? It sounds like your PATH isn't configured to include dart binaries. When you installed it you should have gotten a warning about it and also a line that shows you how to solve it. If you uninstall Melos and install it again the line for how to add it to your PATH will probably show.

**Reply by Killpit** [reply]
macOS, and this is the response to `echo $PATH` and I followed the steps too

`/Users/atatekeli/Downloads/google-cloud-sdk/bin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/Library/Frameworks/Python.framework/Versions/3.11/bin:/Users/atatekeli/.local/share/solana/install/active_release/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/atatekeli/.cargo/bin:/Users/atatekeli/.yarn/bin:/Users/atatekeli/.config/yarn/global/node_modules/.bin:/Library/Frameworks/Python.framework/Versions/3.9/bin:/opt/anaconda3/bin:/Users/atatekeli/Downloads/anaconda3/bin:/Users/atatekeli/opt/anaconda3/condabin:/Library/Frameworks/Python.framework/Versions/3.7/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/Library/Apple/usr/bin:/Applications/VMware Fusion.app/Contents/Public:/usr/local/share/dotnet:~/.dotnet/tools:/usr/local/go/bin:/Users/atatekeli/go/bin:/Users/atatekeli/.avm/bin:/Users/atatekeli/openshift-client-mac-arm64-4.16.13
`

**Reply by Killpit** [reply]
@spydon and should I remove the paths before uninstalling melos

**Reply by spydon** [reply]
As far as I can see you don't have the path to the directory with the dart binaries added, so no you don't have to remove anything, just adding the correct path to the PATH variable is enough.

**Reply by Killpit** [reply]
@spydon these are the paths, and yet I had the same issue to begin with

`/Users/atatekeli/.nvm/versions/node/v20.10.0/bin:/Users/atatekeli/Downloads/google-cloud-sdk/bin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/Library/Frameworks/Python.framework/Versions/3.11/bin:/Users/atatekeli/.local/share/solana/install/active_release/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/atatekeli/.cargo/bin:/Users/atatekeli/.yarn/bin:/Users/atatekeli/.config/yarn/global/node_modules/.bin:/Library/Frameworks/Python.framework/Versions/3.9/bin:/opt/anaconda3/bin:/Users/atatekeli/Downloads/anaconda3/bin:/Users/atatekeli/opt/anaconda3/condabin:/Library/Frameworks/Python.framework/Versions/3.7/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/Library/Apple/usr/bin:/Applications/VMware Fusion.app/Contents/Public:/usr/local/share/dotnet:~/.dotnet/tools:/usr/local/go/bin:/Users/atatekeli/go/bin:/Users/atatekeli/.avm/bin:/Users/atatekeli/openshift-client-mac-arm64-4.16.13: /Users/atatekeli/flutter/flutter/bin:/Users/atatekeli/.foundry/bin: /Users/atatekeli/flutter/flutter/bin:/Users/atatekeli/.pub-cache/bin`

**Reply by spydon** [reply]
What does `find / -name melos 2> /dev/null` say?

**Reply by Killpit** [reply]
@spydon these are what I found when I wrote the command to the terminal

```
/System/Volumes/Data/Users/atatekeli/.pub-cache/bin/melos
/System/Volumes/Data/Users/atatekeli/.pub-cache/hosted/pub.dev/melos-3.0.0/templates/melos
/System/Volumes/Data/Users/atatekeli/.pub-cache/hosted/pub.dev/melos-3.1.0/templates/melos
/System/Volumes/Data/Users/atatekeli/.pub-cache/hosted/pub.dev/melos-6.2.0/templates/melos
/System/Volumes/Data/Users/atatekeli/.pub-cache/global_packages/melos
/Users/atatekeli/.pub-cache/bin/melos
/Users/atatekeli/.pub-cache/hosted/pub.dev/melos-3.0.0/templates/melos
/Users/atatekeli/.pub-cache/hosted/pub.dev/melos-3.1.0/templates/melos
/Users/atatekeli/.pub-cache/hosted/pub.dev/melos-6.2.0/templates/melos
/Users/atatekeli/.pub-cache/global_packages/melos
```

**Reply by Killpit** [reply]
Looks like multiple melos versions, and also fixed the bash configuration on macOS, still need help on how to proceed @spydon
